dll/shellext/shellbtrfs/devices.cpp

```diff
@@ -907,7 +907,11 @@ static INT_PTR CALLBACK stub_DeviceResizeDlgProc(HWND hwndDlg, UINT uMsg, WPARAM
 
 void BtrfsDeviceResize::ShowDialog(HWND hwnd, WCHAR* fn, UINT64 dev_id) {
     this->dev_id = dev_id;
+#ifdef __REACTOS__
+    wcscpy_s(this->fn, _countof(fn), fn);
```

Comment:
`_countof(fn)` is not a meaningful value

Reply:
I used `_countof()` because wcscpy_s wants the size as in characters and not bytes thus `sizeof()` won't work. Or do you mean the whole thing is useless and we should rather use something else than `_countof()`?

Reply:
you probably want `_countof(this->fn)`